src/Microsoft.VisualStudio.AppDesigner/PropPages/PropPageUserControlBase.vb

```diff
@@ -3337,7 +3341,19 @@ NextControl:
         ''' <returns></returns>
         ''' <remarks></remarks>
         Protected Function GetProjectPath() As String
-            Return CStr(GetCommonPropertyValueNative("FullPath")) 'CONSIDER: This won't work for all project types, e.g. ASP.NET when project path is a URL
```

Comment:
No DTE.Object returns OAVsProject which doesn't have return the properties from ConfigurationGeneral. http://index/#Microsoft.VisualStudio.ProjectSystem.VS.Implementation/Package/Automation/VSProject/OAVSProject.cs,0a10689b8d4b8823